website/docs/guides/delivery.mdx

```diff
@@ -0,0 +1,473 @@
+
+```yaml title="linkerd/source.yaml"
+apiVersion: source.toolkit.fluxcd.io/v1beta2
+kind: HelmRepository
+metadata:
+  name: linkerd
+spec:
+  interval: 1h
+  url: https://helm.linkerd.io/stable
+```
+
+Then, to install the latest version of Linkerd, add the following `HelmRelease` manifests to the cluster repository:
+
+```yaml title="linkerd/releases.yaml"
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: linkerd
+spec:
+  interval: 10m
+  chart:
+    spec:
+      chart: linkerd2
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: linkerd
+  install:
+    crds: Create
+  upgrade:
+    crds: CreateReplace
+  valuesFrom:
+    - kind: Secret
+      name: linkerd-certs
+      valuesKey: ca.crt
+      targetPath: identityTrustAnchorsPEM
+    - kind: Secret
+      name: linkerd-certs
+      valuesKey: issuer.crt
+      targetPath: identity.issuer.tls.crtPEM
+    - kind: Secret
+      name: linkerd-certs
+      valuesKey: issuer.key
+      targetPath: identity.issuer.tls.keyPEM
+  values:
+    installNamespace: false
+    identity:
+      issuer:
+        crtExpiry: "2023-07-18T20:00:00Z"
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: linkerd-viz
+spec:
+  interval: 10m
+  dependsOn:
+    - name: linkerd
+  chart:
+    spec:
+      chart: linkerd-viz
+      reconcileStrategy: ChartVersion
+      sourceRef:
+        kind: HelmRepository
+        name: linkerd
+```
+
+The value for the `spec.values.identity.issuer.crtExpiry` field above depends on the parameter value used during the creation of the issuer certificate previously. In this example, it should be set to 1 year from the certificate creation.
+
+Then, add the following file to instruct Kustomize to patch any `Secrets` that are referenced in `HelmRelease` manifests:
+
+```yaml title="linkerd/kustomizeconfig.yaml"
+nameReference:
+  - kind: Secret
+    version: v1
+    fieldSpecs:
+      - path: spec/valuesFrom/name
+        kind: HelmRelease
+```
+
+Finally, add the following Kustomization file that references all the previous files that were added:
+
+```yaml title="linkerd/kustomization.yaml"
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: linkerd
+configurations:
+- kustomizeconfig.yaml
+resources:
+- namespace.yaml
+- source.yaml
+- releases.yaml
+secretGenerator:
+  - name: linkerd-certs
+    files:
+      - ca.crt
+      - issuer.crt
+      - issuer.key
+```
+
+The `secretGenerator` is used to generate Secrets from the generated files. 
+
+The `linkerd` directory in the cluster repository should look like this:
+
+```bash
+> tree linkerd
+linkerd
+├── ca.crt
+├── issuer.crt
+├── issuer.key
+├── kustomization.yaml
+├── kustomizeconfig.yaml
+├── namespace.yaml
+├── releases.yaml
+└── source.yaml
+```
+
+Once Flux reconciles this directory to the cluster, Linkerd should get installed.
```

Comment:
We should add guidance on how to check both Flagger and Linkerd have been installed before proceeding to deploy the canary